amundsen_application/static/js/components/common/LoadingSpinner/styles.scss

```diff
@@ -5,5 +5,6 @@
   height: 100px;
   margin: auto;
   margin-top: 20%;
+  margin-bottom: 20%;
```

Comment:
Added this so it looks slightly better when used in the modal body. Current styles temporarily increase the height of the owner editing section but I didn't invest effort into custom styling the LoadingSpinner because we are supposed to implement the shimmer effect for all loading states.